gst/nnstreamer/nnstreamer_plugin_api_impl.c

```diff
@@ -1661,7 +1661,7 @@ gst_tensors_extra_append_memory_to_buffer (GstBuffer * buffer,
   /* copy last_memory into new_memory */
   if (!gst_memory_map (last_memory, &last_memory_map, GST_MAP_READ)) {
     nns_loge ("Failed to map last memory");
-    gst_memory_unref (new_memory);
+    gst_memory_unref (last_memory);
```

Comment:
no, last_mem from gst_buffer_peek_memory() is not allocated memory. we should release new_mem here (allocated at line 1649)

Reply:
thank you. It's my mistake.